packages/@react-native-windows/cli/README.md

```diff
@@ -37,8 +37,6 @@ Options:
 | `--buildLogDirectory` | Optional directory where MSBuild log files should be stored | [string] |
 | `--info`              | Dump environment information | [boolean] |
 | `--direct-debugging`  | Enable direct debugging on specified port | [number] |
-| `--useWinUI3`         | Targets WinUI 3.0 (Preview) instead of UWP XAML. | [boolean] |
-| `--useHermes`         | Use Hermes instead of Chakra as the JS engine (supported on 0.64+) | [boolean] |
```

Comment:
Why is `--useHermes` being removed?

Reply:
Not removing, just moving to packages/react-native-windows-init/README.md. They were added to the wrong README in  #7788.